backend/src/controllers/action-items.controller.ts

```diff
@@ -25,6 +25,20 @@ class ActionItemsController extends BaseController<IWeeklyActionItems> {
 
     return res.status(httpStatus.OK).send(weeklyActionItems);
   }
+
+  async updateActionItem(req: AuthRequest<IActionItem>, res: Response<void>) {
```

Comment:
Return the updated document, do it using findOneAndUpdate with new: true

Reply:
Why?

Reply:
Because it makes sense when updating a document to get the updated document in return. For simple API usage it gives 2 advantages:
1. I can work with the returned object and I don't have to fetch it again
2. For testing it makes it very easy to see that the returned object really updated

Reply:
Fine, but in the frontend we're just refetchinig after sending the request.

Reply:
It's ok don't worry about the frontend. Just in general for the backend